config/routes.rb

```diff
@@ -41,10 +42,13 @@
   resources :objectives, only: [:destroy]
   resources :public_accounts, param: :token, only: :show, controller: 'public_account_shares'
   resources :feedbacks, only: [:new, :create]
+  resources :sign_in_kids, only: %i[new create]
   resources :archived_accounts, only: %i[index] do
     member do
       get 'restore', action: 'restore'
     end
   end
+
   get '/policy', to: 'policy#show'
+  get '/choose_role', to: 'choose_role#index'
```

Comment:
This will be an entry point for the mobile app. We should not have a link to this page. At least for now. We can reconsider this after testing though.